sr_robot_commander/test/test_robot_commander.py

```diff
@@ -0,0 +1,465 @@
+
+    def test_mvgr_get_and_set_planner_id(self):
+        planner_list = ["BKPIECEkConfigDefault", "ESTkConfigDefault", "KPIECEkConfigDefault",
+                        "LBKPIECEkConfigDefault", "PRMkConfigDefault", "RRTkConfigDefault",
+                        "SBLkConfigDefault", "PRMstarkConfigDefault", "RRTstarkConfigDefault"]
+
+        for id in planner_list:
+            self.robot_commander.set_planner_id(id)
+            self.assertTrue(id == self.robot_commander._move_group_commander.get_planner_id())
+
+    def test_mvgr_get_and_set_planning_time(self):
+        time_test_values = [1, 2, 3, 4, 5]
+        for time in time_test_values:
+            self.robot_commander.set_planning_time(time)
+            self.assertTrue(time == self.robot_commander._move_group_commander.get_planning_time())
+
+    def test_mvgr_set_num_planning_attempts(self):
+        raised = False
+        try:
+            self.robot_commander.set_num_planning_attempts(1)
+        except (NameError, TypeError):
+            raised = True
+        self.assertFalse(raised)
+
+    # def get_end_effector_pose_from_named_state(self, name):
+
+    def test_mvgr_get_end_effector_pose_from_state(self):
+        state = {'ra_shoulder_pan_joint': 0.5157461682721474,
+                 'ra_elbow_joint': 0.6876824920327893,
+                 'ra_wrist_1_joint': -0.7695210732233582,
+                 'ra_wrist_2_joint': 0.2298871642157314,
+                 'ra_shoulder_lift_joint': -0.9569080092786892,
+                 'ra_wrist_3_joint': -0.25991215955733704}
+        rs = RobotState()
+        for key, value in state.items():
+            rs.joint_state.name.append(key)
+            rs.joint_state.position.append(value)
+        pose = self.robot_commander.get_end_effector_pose_from_state(rs)
+
+    def test_mvgr_get_planning_frame(self):
+        raised = False
+        try:
+            self.robot_commander.get_planning_frame()
+        except (NameError, TypeError):
+            raised = True
+        self.assertFalse(raised)
+
+    def test_mvgr_set_and_get_pose_reference_frame(self):
+        reference_frame = "world"
+        self.robot_commander._move_group_commander.set_pose_reference_frame(reference_frame)
+        self.assertTrue(self.robot_commander._move_group_commander.get_pose_reference_frame() == reference_frame)
+
+    def test_get_group_name(self):
+        self.assertTrue(self.robot_commander._name == self.robot_commander.get_group_name())
+
+    def test_refresh_named_targets(self):
+        self.robot_commander.refresh_named_targets()
+        condition_1 = type(self.robot_commander._srdf_names) == list
+        condition_2 = type(self.robot_commander._warehouse_names) == list
+        self.assertTrue(condition_1 and condition_2)
+
+    def test_set_max_velocity_scaling_factor__range_ok(self):
+        raised = False
+        try:
+            self.robot_commander.set_max_velocity_scaling_factor(0.2)
+        except Exception:
```

Comment:
i think you don't need to wrap up this in a try catch, the unit test will fail if the exception is thrown.

it can just be self.robot_commander.set_max_velocity_scaling_factor(0.2). and should pass. maybe add a comment like /*expected no exception to be raised*/. Up to you tho, not entirely sure about this one :)

Reply:
left bare method call